doc/requirements.in

```
@@ -2,4 +2,4 @@ breathe
 myst-parser
 sphinx
 sphinx_rtd_theme
-sphinx_copybutton
+sphinx_copybutton
```

```suggestion
sphinx_copybutton
```
Simply to not touch this file in this PR.